src/schema-builder/table/TableColumn.ts

```diff
@@ -78,6 +78,12 @@ export class TableColumn {
      */
     width?: number
 
+    /**
+     * Vector dimensions. Used only for vector type.
+     * For example type = "vector" and dimensions = 3 means that we will create a column with type vector(3).
+     */
+    dimensions?: number
```

Comment:
Can we just use `length` instead? Basically the number of dimensions of a vector is its length, so there's no need to add something separate for it.